import-export-cli/box/resources/kubernetes_resources/api_cr.yaml

```diff
@@ -1,11 +1,6 @@
-apiVersion: wso2.com/v1alpha1
+apiVersion: wso2.com/v1alpha2
```

Comment:
Shall we run **go run tools/gen.go** and see whether which .md or .sh files get changed. You can commit those changes as well. (Probably, **apictl.md** and **apictl_bash_completions.sh** will get changed)

Reply:
Fixed in d7c19b7176356c17471a83933a9a1879d9471a36